Entities/Timetable.cs

```diff
@@ -1,23 +1,55 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace FRITeam.Swapify.Entities
 {
-    public class Timetable: BaseEntity
+    public class Timetable : BaseEntity
     {
-        public List<Block> Blocks { get; set; }
+        /// <summary>
+        /// !!! Do not use this property from outside this class,
+        /// public is just because of mongodb mapper !!
+        /// </summary>
+        public List<Block> _blocks { get; set; }
```

Comment:
OK, I dont like this "hack" with RegistrationClassMap :( but OK. I am not sure about place where should be this piece of code: 
 _BsonClassMap.RegisterClassMap<Timetable>(cm =>
            {
                cm.AutoMap();
                cm.MapField("_blocks").SetElementName("Blocks");
            });_

Do you have any suggestion?

Reply:
Why do you don't like?
It should be called ASAP after application start. So it could be some static class called in some intitialisation of the server.

Reply:
Ok